OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -650,9 +662,15 @@ void UpdatePlayerList()
 					if (template == null || template.Id != nonEditableSpectatorTemplate.Id)
 						template = nonEditableSpectatorTemplate.Clone();
 
-					LobbyUtils.SetupNameWidget(template, null, client);
-					LobbyUtils.SetupKickWidget(template, null, client, orderManager, lobby,
-						() => panel = PanelType.Kick, () => panel = PanelType.Players);
+					if (isHost)
+					{
+						LobbyUtils.SetupPlayerActionWidget(template, null, client, orderManager, lobby,
+								() => panel = PanelType.Kick, () => panel = PanelType.Players);
+					}
+					else
+					{
```

Comment:
Style nit: omit braces around single line blocks.  This may apply to other places in this PR too.